extensions/wikia/Search/js/WikiaSearch.js

```diff
@@ -1,6 +1,54 @@
 (function($, window) {
 
 require(['search-tracking', 'uuid', 'wikia.trackingOptIn'], function(searchTracking, uuid, trackingOptIn) {
+	var resultClickTrackerFactory = function (type, idGenerator, filtersProvider) {
+		return function(clickedElement) {
+			var queryparams = new URL(window.location).searchParams;
+			var query = queryparams.get('search') || queryparams.get('query');
+
+			if (!query) {
+				return;
```

Comment:
I'm not sure this is right, because on the search page, the users can enter an empty query, and clicking something would mean that the user got something valuable, right?

Reply:
It was like that (probably because old search had similar condition on backend), but you're right - I will change that.
Thanks!

Reply:
So, will ya? :)